osbuild/buildroot.py

```diff
@@ -86,6 +90,30 @@ def run(self, argv, binds=None, readonly_binds=None, **kwargs):
             # wants to be able create devices nodes, so allow that
             loopback_allow += "m"
 
+        # make osbuild API-calls accessible to the container
+        nspawn_ro_binds.append(f"{self.api}:/run/osbuild/api")
+
+        # We want to execute our stages and other scripts in the container. So
+        # far, we do not install osbuild as a package in the container, but
+        # provide it from the outside. Therefore, we need to provide `libdir`
+        # via bind-mount. Furthermore, a system-installed `libdir` has the
+        # python packages separate in `site-packages`, so we need to bind-mount
+        # them as well.
+        # In the future, we want to work towards mandating an osbuild package to
```

Comment:
I am not entirely sure that this is what we want long-term. Something to discuss.